content/ADR-172-rentals.md

```diff
@@ -0,0 +1,88 @@
+
+## Context, Reach & Prioritization
+
+Rentals are a great way to earn passive income over assets. This will benefit owners that currently hold Land but don't know or want to build on it, creators that might find it expensive to buy Land and deploy scenes on it, and the whole of Genesis City as more and more creators will have access to populate and give it more life.
+
+Being able to rent your own Land was something the community has been asking for a while now. Rental solutions do exist, from dapps and smart contracts deployed by the community, to rentals being handled off-chain in a peer-2-peer manner.
+
+However, unofficial protocols and off-chain solutions might not take in consideration the best practices for security to protect users as well as Decentraland's most precious assets. That is why Decentraland has developed their own protocol.
+
+### How it works?
+
+The [Rentals Smart Contract](https://etherscan.io/address/0x3a1469499d0be105d4f77045ca403a5f6dc2f3f5#code) allow users to accept Rental Listings and Offers. 
+
+These can be created off chain by signing them with your wallet, and storing the data and signature on a place that can be consumed by an interested user.
+
+The benefit of using off-chain signatures, is that users can create Listings and Offers without the need of paying for a transaction.
+
+The Land owner can accept an Offer by calling the `acceptOffer` function or safely transferring the Land to the Rentals contract with the offer data and signature. 
+
+A user that is interested in renting a Land can accept a Listing by calling the `acceptListing` function on the contract with the Listing data and signature.
+
+Successfully accepting any of these will start a Rental that will last as long as the Listing/Offer stipulated, MANA will be transferred from the tenant to the lessor, the rented Land will be transferred to the Rentals contract, and the tenant or any address determined by the tenant will be set as update operator of the Land to be able to deploy scenes.
+
+### Why is the Land transferred to the Rentals contract?
+
+This is to limit what the original owner of the asset can do with the Land, by transferring the ownership to the Rentals contract, the owner cannot interact with the asset directly anymore and have to use a set of functions defined by the Rentals smart contract to do so.
+
+For example, Once a rental starts, the owner can't do anything until the rental period ends. In that case, they can `claim` the Land back, rent it again the same way as before or set a different update operator to overwrite the one set by the tenant and prevent it's further use.
+
+### What happens when the rental ends?
+
+Nothing actively happens, if nothing is done by the owner, the update operator defined by the tenant can still deploy scenes. The owner can claim the Land back, rent it again or change the update operator to prevent the tenant from using it.
```

Comment:
```suggestion
Nothing actively happens. Unless the owner changes something, the update operator defined by the tenant can still deploy scenes. The owner can claim the Land back, rent it again or change the update operator to prevent the tenant from using it.
```